meson_options.txt

```diff
@@ -17,6 +17,7 @@ option('plugin_amt', type : 'boolean', value : true, description : 'enable Intel
 option('plugin_dell', type : 'boolean', value : true, description : 'enable Dell-specific support')
 option('plugin_dummy', type : 'boolean', value : false, description : 'enable the dummy device')
 option('plugin_emmc', type : 'boolean', value : true, description : 'enable eMMC support')
+option('plugin_logitech_bulkcontroller', type : 'boolean', value : true, description : 'enable Logitech Rally Bar support')
```

Comment:
Surely `enable Logitech Bulk Controller support`? Rally Bar is just one implementation, no?

Reply:
Changed